core/tx_pool.go

```diff
@@ -55,6 +56,15 @@ var (
 	evictionInterval     = time.Minute   // Time interval to check for evictable transactions
 )
 
+var (
+	// Metrics
+	pendingDiscardMeter    = metrics.NewMeter("core/txpool/transactions/pending/discard")
+	pendingReplaceMeter    = metrics.NewMeter("core/txpool/transactions/pending/replace")
+	queuedDiscardMeter     = metrics.NewMeter("core/txpool/transactions/queued/discard")
+	queuedReplaceMeter     = metrics.NewMeter("core/txpool/transactions/queued/replace")
+	invalidTxMeter         = metrics.NewMeter("core/txpool/transactions/invalid")
```

Comment:
I'd suggest replacing the `core/txpool/transactions` namespace with only `txpool`. There's really no reason to have such long names imho. I've tried to group metrics according to certain namespace, but no need to be too verbose about them, as long as they are obvious.

Here are the current ones https://gist.github.com/karalabe/552f7d6f7204387107615d38aeeae7ac